### PR TITLE
Fix/added hint icon to expiry date

### PR DESCRIPTION
**Pull request by jobsturm**
## Summary
Added expiry date icon as hint to the expiry date fields

## Tested scenarios
- [x] Tested in the Playground
- [x] Tested when field is invalid
- [x] Tested when field is valid
- [x] Tested when field is empty

### CODE REVIEW

File: packages/lib/src/components/Card/components/CardInput/components/StoredCardFields.tsx [hunk 1]
```diff
@@ -41,6 +41,13 @@ export default function StoredCardFields({
                     >
                         {expiryMonth} / {expiryYear}
                     </div>
+                    <div className="adyen-checkout__field__exp-date_hint_wrapper">
```
Comment (sponglord) [inline packages/lib/src/components/Card/components/CardInput/components/StoredCardFields.tsx:44]: Do we need the date icon in the Stored Card?

Comment (jobsturm) [reply packages/lib/src/components/Card/components/CardInput/components/StoredCardFields.tsx]: It isn't, removed it!

File: packages/lib/src/components/Card/components/CardInput/components/ExpirationDate.tsx [hunk 2]
```diff
@@ -56,6 +57,19 @@ export default function ExpirationDate(props: ExpirationDateProps) {
...
+                    'adyen-checkout__field__exp-date_hint_wrapper--hidden': error || isValid,
+                }
+            )}>
+                <img
+                    src="https://checkoutshopper-test.adyen.com/checkoutshopper/images/components/expiry_date_hint.svg"
+                    className="adyen-checkout__field__exp-date_hint"
+                    alt={fieldLabel}
+                />
+            </div>
         </Field>
```
Comment (ribeiroguilherme) [inline packages/lib/src/components/Card/components/CardInput/components/ExpirationDate.tsx:73]: Why aren't we implementing the same way as the CVC hint image in this case?

Comment (sponglord) [reply packages/lib/src/components/Card/components/CardInput/components/ExpirationDate.tsx:73]: I believe it's to do with the fact that the cvc icon animates, so is an inline svg. Whilst the date icon does not - so we load it from the server

**Review by sponglord — APPROVED** [review]

**Pull request merged**